### PR TITLE
01 android build

### PR DESCRIPTION
**Pull request by ShawnaRStaff**
Android build
updated read me
.env examples

### CODE REVIEW

File: MentalHealthAppFrontEnd/api/auth.ts [hunk 1]
```diff
@@ -27,8 +27,9 @@ export const login = async (email: string, password: string) => {
     });
 
     const data = await response.json();
+    console.log("data", data);
```
Comment (sanckh) [inline MentalHealthAppFrontEnd/api/auth.ts]: remove console.log

File: backend/src/firebase_options.ts [hunk 2]
```diff
@@ -15,12 +15,12 @@ if (process.env.NODE_ENV !== 'production') {
 
 
 // Parse the service account JSON from the environment variable
-const serviceAccountPath = process.env.GOOGLE_APPLICATION_CREDENTIALS;
+const serviceAccountPath = "F:/web_projects/mentalhealthapp/backend/src/serviceAccountKey.json";
```
Comment (sanckh) [inline backend/src/firebase_options.ts]: change this back to using the google application credentials

File: backend/src/firebase_options.ts [hunk 3]
```diff
@@ -15,12 +15,12 @@ if (process.env.NODE_ENV !== 'production') {
...
 
-if (!serviceAccountPath || !fs.existsSync(serviceAccountPath)) {
-  console.error('Service account key file not found!');
-  process.exit(1); // Stop the process if the key is missing
-}
+// if (!serviceAccountPath || !fs.existsSync(serviceAccountPath)) {
+//   console.error('Service account key file not found!');
+//   process.exit(1); // Stop the process if the key is missing
+// }
```
Comment (sanckh) [inline backend/src/firebase_options.ts]: id remove these comments because it seems to be an issue with your machine currently

Comment (ShawnaRStaff) [reply backend/src/firebase_options.ts]: No need to I have resolved this issue and typescript has a fit without the null check

**Review by sanckh — APPROVED** [review]

**Pull request merged**